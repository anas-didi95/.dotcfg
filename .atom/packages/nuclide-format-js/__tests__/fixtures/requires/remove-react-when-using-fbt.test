const React = require('React');

const label = <fbt>lorem ipsum <fbt:param>{"dolor sit amet"}</fbt:param></fbt>;
