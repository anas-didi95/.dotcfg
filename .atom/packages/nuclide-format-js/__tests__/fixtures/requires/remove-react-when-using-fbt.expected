const fbt = require('fbt');

const label = <fbt>lorem ipsum <fbt:param>{"dolor sit amet"}</fbt:param></fbt>;
